### PR TITLE
edited the distributors schema to account for tokens

### PR DESCRIPTION
**Pull request by ryannamgung**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -200,75 +201,78 @@ GEM
...
     unf (0.1.4)
       unf_ext
     unf_ext (0.0.7.7)
     unicode-display_width (1.7.0)
-    websocket-driver (0.7.1)
+    websocket-driver (0.7.3)
       websocket-extensions (>= 0.1.0)
     websocket-extensions (0.1.5)
-    zeitwerk (2.3.0)
+    zeitwerk (2.4.1)
```
Comment (justintmckibben) [inline Gemfile.lock:275]: What is all of this? Might wanna revert this to master

File: db/migrate/20201116222055_add_tokens_to_distributors.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class AddTokensToDistributors < ActiveRecord::Migration[6.0]
+  def change
+    add_column :distributors, :gift_card_login_token, :string
+    add_column :distributors, :gift_card_login_expires_at, :datetime
+    add_column :distributors, :gift_card_login_state, :string, default: 'new', null: false
```
Comment (justintmckibben) [inline db/migrate/20201116222055_add_tokens_to_distributors.rb:5]: We should make this enum a number vs a string

Comment (justintmckibben) [reply db/migrate/20201116222055_add_tokens_to_distributors.rb:5]: Also add the states in the model file

**Review by justintmckibben — CHANGES_REQUESTED** [review]

**Comment by jacoblb64** [comment]
Closing this as it's been way out of date

**Pull request closed without merging**